### ISSUE TITLE
Extra backups not being deleted

### ISSUE THREAD

**Issue by katmoody** [humanmade/backupwordpress-pro-gdrive#46]
Though the "max backups" is set to one number, the backups aren't being kept to that minimum in Google Drive - it's keeping them until the person deletes them themselves. 

I've heard this before, but just got a new ticket in so wanted to log it. 

Latest ticket: https://app.intercom.io/a/apps/7f1l4qyq/inbox/all/conversations/4871686194

> Also, Google Drive is STILL keeping more backups than I have the number set to. I STILL have to manually go in to Google Drive and remove old backups.

**Comment by katmoody** [comment]
I have access to the site for testing on this one - anything specific that would be worth testing?

**Comment by pdewouters** [comment]
if you setup google drive backups on your test site, does it delete old backups?

**Comment by roborourke** [comment]
Maybe make sure they have the latest versions of everything if you haven't already @katmoody. I just tested locally and never get more than the max num in my drive folder.

Will see what happens when I change the value up & down too.

**Comment by roborourke** [comment]
Yep confirmed, when changing the number it works fine. Doesn't remove existing ones when changing down until the next backup runs.

@katmoody have you asked to re-authenticate? Might be a token permission issue depending on when it was originally set up.
